manubot/cite/csl_item.py

```diff
@@ -149,7 +150,11 @@ def clean(self, prune: bool = True):
             self.validate_against_schema()
         return self
 
-    def set_date(self, date, variable="issued"):
+    def set_date(
+        self,
+        date: Union[None, str, datetime.date, datetime.datetime],
+        variable="issued",
```

Comment:
I didn't catch this before merging, but should this be `variable: str = "issued",` to match line 168?

Reply:
yeah good catch.

Reply:
added in https://github.com/manubot/manubot/pull/252